ts/client/src/client.ts

```diff
@@ -937,44 +902,45 @@ export class MangoClient {
     ownerPk: PublicKey,
     loadSerum3Oo = false,
   ): Promise<MangoAccount[]> {
-    const anchroHeaderFilter: {
+    const discriminatorMemcmp: {
       offset: number;
       bytes: string;
     } = this.program.account.mangoAccount.coder.accounts.memcmp(
       this.program.account.mangoAccount['_idlAccount'].name,
       undefined,
     );
 
-    const accounts = (
-      await this.program.provider.connection.getProgramAccounts(
-        this.programId,
-        {
-          filters: [
-            {
-              memcmp: {
-                bytes: anchroHeaderFilter.bytes,
-                offset: anchroHeaderFilter.offset,
+    const accounts = await Promise.all(
+      (
+        await this.program.provider.connection.getProgramAccounts(
+          this.programId,
+          {
+            filters: [
+              {
+                memcmp: {
```

Comment:
memcmp: discriminatorMemcmp didn't do it?

Reply:
sorry, could you rephrase please?

Reply:
I mean, do you need to do
```
memcmp: {
    bytes: a.bytes,
    offset: a.offset,
}
```
? wouldn't `memcmp: a` work?